specs/merge/validator.md

```diff
@@ -66,5 +66,6 @@ def get_execution_payload(state: BeaconState) -> ExecutionPayload:
 
```

Comment:
(couldn't comment on to L65 on GitHub, commenting on L66 instead)

L65 also needs `timestamp` argument for calling `produce_execution_payload`.

Reply:
Well spotted 👍 Fixed now